src/libmongoc/tests/test-mongoc-background-monitoring.c

```diff
@@ -253,7 +296,11 @@ tf_destroy (test_fixture_t *tf)
       bson_mutex_unlock (&_tf->mutex);                     \
    } while (0);
 
-#define WAIT _mongoc_usleep (10 * 1000);
+/* Wait for two periods of the faster heartbeat.
```

Comment:
What does "two periods of the faster heartbeat" refer to? If this pertains to `min_heartbeat_frequency_msec` being changed to `10` earlier, should we instead use a constant for that and change this to `2 * CONSTANT * 1000`?